kura/emulator/org.eclipse.kura.emulator/src/main/resources/kura.properties

```diff
@@ -34,6 +36,9 @@ kura.tmp=/tmp/kura/tmp
 kura.snapshots=/tmp/kura/snapshots
 kura.snapshots.count=10
 kura.have.net.admin=false
+# kura.keystore.password=password
+# kura.ssl.keyStorePassword=password
+kura.wifi.top.channel=11
```

Comment:
Why do we need to set kura.wifi.top.channel in the emulator?

Reply:
In this case is just to align to the configurations we actually have.
For example, now the emulator configuration is close to the nn profiles we have. This will simplify a lot the maintenance.